requirements.txt

```
@@ -6,7 +6,7 @@ h5py==3.6.0
 idna==2.10
 kiwisolver==1.3.2
 matplotlib==3.5.0
-numpy==1.21.3
+numpy==1.21.0
```

If we decide to proceed, this change seems unnecessary 
```suggestion
numpy==1.21.3
```